content-elements/form/form-field/prototype/form-field.js

```diff
@@ -46,22 +46,101 @@ Alpine.data('formField', () => ({
       this.requiredErrorElement.style.display = "none";
     }
     this._validateMailInput();
+    this._validateNumberInput();
+    this._validateTextInput();
+    this._validateDateTimeInput();
+  },
+
+  _validateNumberInput() {
+    if (this.inputEl.type === 'number') {
+      let inputValue = parseInt(this.inputEl.value);
+      let minValue = parseInt(this.inputEl.min);
+      let maxValue = parseInt(this.inputEl.max);
+      let valid = minValue <= inputValue && inputValue <= maxValue && inputValue != '';
+      // don't check required validation
+      if (this.inputEl.value == '' || this.inputEl.value == this.inputEl.defaultValue) {
```

Comment:
`this.inputEl.value == ''` - is this case being caught already in case of catching the general required input or how is this  handled?